File: app/src/main/java/com/lamnt/foodorder/view/fragment/SplashFragment.java
package com.lamnt.foodorder.view.fragment;

import android.os.Bundle;
import android.os.Handler;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ProgressBar;

import androidx.fragment.app.Fragment;

import com.lamnt.foodorder.R;
import com.lamnt.foodorder.utils.FragmentUtil;
import com.lamnt.foodorder.view.fragment.base.BaseFragment;

import butterknife.BindView;

/**
 * A simple {@link Fragment} subclass.
 */
public class SplashFragment extends BaseFragment {
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        layoutRes = R.layout.fragment_splash;
        return super.onCreateView(inflater, container, savedInstanceState);
    }

    @Override
    protected void unit(View v) {
        new Handler().postDelayed(() ->
                        FragmentUtil.replaceFragment(
                                mActivity,
                                R.id.frame_splash_container,
                                new IntroFragment(),
                                true),
                4000);
    }

    @Override
    protected boolean isShowNotificationIcon() {
        return false;
    }

    @Override
    protected boolean isShowSearchIcon() {
        return false;
    }

    @Override
    protected boolean isShowBottomNav() {
        return false;
    }
}

File: app/src/main/java/com/lamnt/foodorder/view/staff/fragment/intro/SplashFragment.kt
package com.lamnt.foodorder.view.staff.fragment.intro

import android.annotation.SuppressLint
import android.os.Handler
import android.os.Looper
import com.lamnt.foodorder.R
import com.lamnt.foodorder.databinding.FragmentSplashBinding
import com.lamnt.foodorder.utils.ActivityUtil.replaceFragment
import com.lamnt.foodorder.view.base.BaseFragment
import kotlinx.android.synthetic.main.fragment_splash.*

/**
 * A simple [Fragment] subclass.
 */
class SplashFragment : BaseFragment<FragmentSplashBinding>() {
    override fun getLayoutRes(): Int = R.layout.fragment_splash

    override fun setTitle(): Int = 0

    @SuppressLint("ResourceType")
    override fun initViews() {
        viewBinding.imgType.setAnimation(R.raw.delivery_animate)
        Handler(Looper.myLooper()!!).postDelayed({
            replaceFragment(
                activity,
                R.id.frame_splash_container,
                IntroFragment(),
                true)
        },
            4000)
    }

    override fun initDataBinding() {

    }

    override fun initData() {

    }
}